report/2.Analysis/EducationalMiniComputers.tex

```diff
@@ -0,0 +1,30 @@
+\section{Educational mini computers}
+
+Apart from the Arduino board, a lot of other mini computers have been created with educational purposes in mind.
+To figure out why these "mini computers" are so popular for technical learning, some of the most popular ones were analysed.
+This should help to figure out which functionality the HCL programming language should include in order to comply with the intend of the mini computers.
```

Comment:
* "Intend" seems a bit to strong here. unless the intend is stated explicitly.

Reply:
The added source explicitly states that one of the purposes of the SBCs were educational. I rephrased to "educational intend" to indicate, that it is the educational intend we are talking about.